cmd/influx/organization_test.go

```diff
@@ -411,6 +431,8 @@ func TestCmdOrg(t *testing.T) {
 			}
 
 			testMemberFn(t, "list", cmdFn, tests...)
+			testMemberFn(t, "ls", cmdFn, tests[:0]...)
```

Comment:
I don't think this is testing anything here :thinking: , did you mean to put `testMemberFn(t, "ls", tests...)`?

`tests[:0] == empty slice`

Reply:
I wanted the first test, since running it against every test seems unnecessary. Fixing.